nautobot/dcim/views.py

```diff
@@ -136,6 +136,11 @@ def post(self, request):
         )
 
 
+class BaseDeviceComponentsBulkRenameView(generic.BulkRenameView):
+    def get_selected_objects_parents_name(self, selected_objects):
+        return selected_objects.first().parent.name
+
+
```

Comment:
This feels like it *should* be possible to include on the base `BulkRenameView` class at this point - that view is AFAIK only used for device-components + VMInterface. I don't think this needs to be a separate subclass.